### ISSUE TITLE
SLU16 - Workflow - Proposed changes

### ISSUE THREAD

**Issue by cimendes** [LDSSA/curriculum-development#35]
# Proposed changes to SLU16 Workflow

## Context

Currently this is the only small learning unit to cover pipelines and custom sklearn objects. This last part was severely lacking, with only one example of a custom estimator

## Detailed Description

Add custom sklearn objects to the curriculum, with emphasis on custom estimators 

Belongs to Specialization 1 - SLU16 (Workflow9

## Possible Implementation

Current Learning materials already reflect this change, but can be improved in future editions.

**Comment by UrbanoFonseca** [comment]
I agree that covering custom estimators would be super useful, specially for later developments and not necessarily the initial stages of the bootcamp. In practice, I find custom transformers to be much useful than custom estimators for students to learn, as you (most likely) will need to develop better data processing tools for your data instead of new models/algorithmia. 
**Unless** we are talking about very specific domain knowledge you may have about a dataset (e.g. business rules), which a Predictor seems more appropriate. 

@cimendes You have the unit for next batch as well. Feel free to add (or not) a practical example of the Custom Estimators, my only advice is to keep it simple as the SLU already contains a lot of new info for the students.

**Comment by cimendes** [comment]
Given the size of the unit, completely replacing custom estimators (that I've never used) with custom transformer (that I've used A LOT) seems like a good compromise :)
